src/backends/kafka/config.rs

```diff
@@ -0,0 +1,106 @@
+use rdkafka::config::ClientConfig as RdKafkaConfig;
+use std::collections::HashMap;
+
+const DEFAULT_QUEUED_MAX_MESSAGE_KBYTES: u32 = 50_000;
+const DEFAULT_QUEUED_MIN_MESSAGES: u32 = 10_000;
+
+#[derive(Debug, Clone)]
+pub struct KafkaConfig {
+    config_map: HashMap<String, String>,
```

Comment:
Do you expect to make this more explicit and typed or is it going to stay a HashMap?
If this is just a wrapper around a HashMap, what is the reason to create a `struct` over just a type alias ?

Reply:
Well, because KafkaConfig has a bunch of methods on it to create consumer and producer configs with sensible defaults, and can be easily converted to a RdKafkaConfig. I think a struct is needed for that.

Reply:
I think in rust you can get away with a type alias and making the functions and the conversion based on that.